src/appTheme.ts

```diff
@@ -21,8 +21,8 @@ export const appTheme: ThemeConfig = {
     colorSuccess: 'var(--j2-color-success)',
     colorWarning: 'var(--j2-color-warning)',
     colorInfo: 'var(--j2-color-info)',
-    colorError: 'var(--j2-color-error)',
     colorLink: 'var(--j2-color-link)',
+    colorError: '#bf6c6f',
```

Comment:
Ahh I see I see, that makes sense to me! Maybe we can make a bug ticket for it? Seems like it's probably a pretty isolated issue at least

Reply:
My hunch is that when defining these base colors (e.g `colorError`, `colorPrimary`), antd computes the values for the other variations (e.g `colorErrorBg`). I imagine that since whatever algorithm does not now how to deal with the CSS variable string, it defaults to `#000000`.